<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayList</class>
 <widget class="QWidget" name="PlayList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(4, 2, 57,190);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>39</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>320</width>
       <height>39</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(0, 170, 255),stop:1  rgb(0, 255, 255));</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="prePageButton">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>39</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>39</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);
color:white;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;播放列表选择&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(0, 170, 255),stop:1  rgb(0, 255, 255));
border-radius:0px;
border:0px soild red;
}
QPushButton:pressed{
border-radius:5px;
	border:0px solid red;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(0, 170, 255),stop:1  rgb(0, 255, 255));
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(0, 170, 255),stop:1  rgb(0, 255, 255));

border-radius:5px;
	border:0px solid red;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/playerIcons/reloadPlay.png</normalon>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-image: url(:/images/playerIcons/playlistdelete.png);
border:0px;
}
QPushButton:pressed{
border-image:url(:/images/playerIcons/playlistdeletePress.png);
border:0px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>410</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>320</width>
       <height>410</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="mediaFilePositionSelect">
      <widget class="QToolButton" name="UsbListButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>181</width>
         <height>81</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>181</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>181</width>
         <height>81</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton{
color:rgba(255,255,255,255);
background-color:rgb(110, 84, 177);
border:none;
padding-top:5px;
padding-bottom:5px;
border:1px solid red;
border-radius:10px;
}
QToolButton:pressed{
	background-color: rgb(7, 255, 139);
}
QToolButton:hover{
	background-color: rgb(7, 255, 139);
}
QToolButton:released{
	background-color:rgb(110, 84, 177);
}</string>
       </property>
       <property name="text">
        <string>U盘媒体列表</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/images/playerIcons/usb.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="favoriteListButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>220</y>
         <width>181</width>
         <height>81</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>181</width>
         <height>81</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>181</width>
         <height>81</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton{
color:rgba(255,255,255,255);
background-color:rgb(110, 84, 177);
border:none;
padding-top:5px;
padding-bottom:5px;
border:1px solid red;
border-radius:10px;
}
QToolButton:pressed{
	background-color: rgb(7, 255, 139);
}
QToolButton:hover{
	background-color: rgb(7, 255, 139);
}
QToolButton:released{
	background-color:rgb(110, 84, 177);
}</string>
       </property>
       <property name="text">
        <string>收藏列表</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/images/playerIcons/favorite.png</normalon>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="udiskMusicList">
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>320</width>
         <height>411</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>411</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>411</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget{
	background-color: rgb(170, 255, 255);
	alternate-background-color: rgb(118, 177, 177);
	border:none;
	border-top: 4px solid  qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(191, 191, 191, 255), stop:1 		rgba(255, 255, 255, 255));
}
QTableWidget:pressed{
	background-color: skyblue;
}
QScrollBar:vertical {
      border: none;
 	  background-color: rgba(255, 255, 255);
      width: 4px;
      margin: 0px 0 0px 0;
  }
  QScrollBar::handle:vertical {
	 border: none;
     border-radius:2px;
      background: rgba(154, 154, 154, 150);
      min-height: 15px;
  }
QScrollBar::sub-line:vertical {
      border: none;
      height: 0px;
      subcontrol-position: top;
      subcontrol-origin: margin;
  }
QScrollBar::add-line:vertical {
      border: none;
      height: 0px;
      subcontrol-position: bottom;
      subcontrol-origin: margin;
  }
  QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	border:none;
      width: 0px;
      height: 0px;
  }
  QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
  }</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScrollMargin">
        <number>20</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::NoPen</enum>
       </property>
       <property name="rowCount">
        <number>100</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>160</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
      </widget>
     </widget>
     <widget class="QWidget" name="favoriteMusicList">
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>320</width>
         <height>411</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>411</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>411</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget{
	background-color: rgb(170, 255, 255);
	alternate-background-color: rgb(118, 177, 177);
	border:none;
	border-top: 4px solid  qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(191, 191, 191, 255), stop:1 		rgba(255, 255, 255, 255));
}
QTableWidget:pressed{
	background-color: skyblue;
}
QScrollBar:vertical {
      border: none;
 	  background-color: rgba(255, 255, 255);
      width: 4px;
      margin: 0px 0 0px 0;
  }
  QScrollBar::handle:vertical {
	 border: none;
     border-radius:2px;
      background: rgba(154, 154, 154, 150);
      min-height: 15px;
  }
QScrollBar::sub-line:vertical {
      border: none;
      height: 0px;
      subcontrol-position: top;
      subcontrol-origin: margin;
  }
QScrollBar::add-line:vertical {
      border: none;
      height: 0px;
      subcontrol-position: bottom;
      subcontrol-origin: margin;
  }
  QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	border:none;
      width: 0px;
      height: 0px;
  }
  QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
  }</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScrollMargin">
        <number>20</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::NoPen</enum>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>160</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column/>
       <column/>
       <column/>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>320</width>
       <height>21</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(45, 189, 250);
color:white;</string>
     </property>
     <property name="text">
      <string>状态：</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
